named_entity_recognition/README.md

```diff
@@ -0,0 +1,35 @@
+# Named Entity Recognition with Kubeflow and Keras 
+
+This example demonstrates how you can use Kubeflow to train and deploy a Keras model with a custom prediction routine. 
+
+## Goals
+
+* Demonstrate how to use Keras only models
+* Demonstrate how to train a Named Entity Recognition model
+* Demonstrate how to deploy a Keras model to AI Platform
+* Demonstrate how to use a custom prediction routine
+* Demonstrate how to use Kubeflow metrics
```

Comment:
Kubeflow metrics & visualizations - are these referring to Tensorboard?

Reply:
Visualizations refer to the TensorBoard visualization used in the training component.
Metric refer to the metrics which are displayed as result of the experiments UI (I will add more explanation)

Reply:
✓ done